pkg/controller/podgc/gc_controller_test.go

```diff
@@ -317,11 +324,16 @@ func TestGCOrphaned(t *testing.T) {
 
 	for _, test := range testCases {
 		t.Run(test.name, func(t *testing.T) {
-			nodeList := &v1.NodeList{}
+			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PodDisruptionConditions, test.enablePodDisruptionConditions)()
+			nodes := make([]*v1.Node, 0, len(test.initialClientNodes))
 			for _, node := range test.initialClientNodes {
-				nodeList.Items = append(nodeList.Items, *node)
+				nodes = append(nodes, node)
 			}
-			client := fake.NewSimpleClientset(nodeList)
+			pods := make([]*v1.Pod, 0, len(test.pods))
+			for _, pod := range test.pods {
+				pods = append(pods, pod)
+			}
+			client := setupNewSimpleClient(nodes, pods)
```

Comment:
I've done the same question here, is just a curiosity https://github.com/kubernetes/kubernetes/pull/110959/files#r935230821 , but if you are confident with the test behavior you can disregard it

Reply:
See my response to the other place: https://github.com/kubernetes/kubernetes/pull/110959#discussion_r935309900